.github/workflows/format-lint.yml

```diff
@@ -0,0 +1,32 @@
+name: Format
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  format_job:
+    name: Format Job
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v4
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version-file: 'package.json'
+
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Run Prettier formatting
+      run: npm run format
+
+# TODO: Test this Action so it actually saves the changes it makes. Maybe run git commit or use external lint action
```

Comment:
Ei järkeä muotoilla CI:ssä, linttaus sen sijaan. Ajatuksena on, että muotoilu tapahtuisi lokaalisti ja CI:ssä varmistetaan, että tyyli vastaa määritettyjä sääntöjä.

Reply:
ok eli formatointi yksittäisten tyypien vastuulla ja lintteri inisee huonosta koodista CIssä

Reply:
Just näin